workspace/utilities/com/modules/module.xsl

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this ?

Reply:
Starting point for all modules components. Keeps everything normalized and easy to maintain.

Reply:
Is it imported by default ?

Maybe we should put "templates" in another folder ?

Reply:
Every modules calls this template so yes, it needs to be imported by default for all modules to work. I'm not against a new folder, but I also think it being in the "Modules" folder works too. Maybe the kind of thing we can think about after this PR is succesfull, hehe!

Reply:
I'll keep it for now, but this is a performance hit. Do not be afraid to copy paste things. It it sometimes more performant and more flexible than having layers and layers of template that do not have a clear responsibility.